packages/core-plugin-api/src/extensions/extensions.tsx

```diff
@@ -256,6 +261,7 @@ export function createReactExtension<
       };
 
       attachComponentData(Result, 'core.plugin', plugin);
+      attachComponentData(Result, 'name', name);
```

Comment:
I'm don't know the answer to this, but should it be namespaced with a prefix like `homepage.name`?

Reply:
Great question, I don't mind it either way. Changing it will have some work to be done to the grid but it's not bad. Any opinions @freben @benjdlambert @tudi2d ?

Reply:
Changed this to `homepage.name`